packages/node/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@vercel/node",
-  "version": "1.12.2-canary.4",
+  "name": "vercel-plugin-node",
```

Comment:
I think we should leave `@vercel/node` as-is and instead introduce a new package `vercel-plugin-node`.